build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+dotnet build --configuration Release
+rm -rf ccgateway/bin/Release/netcoreapp3.1/plugins
+mkdir ccgateway/bin/Release/netcoreapp3.1/plugins
+cp ccgateway/plugins/*.dll ccgateway/bin/Release/netcoreapp3.1/plugins/
+docker build --tag creditcoin-gateway ccgateway/bin/Release/netcoreapp3.1/.
```

Comment:
`.github/workflows/dotnet.yml` should at least call this script to make sure we don't result in errors.

Reply:
Implement build purely via dockerfile and updated github actions to test docker build